extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -143,10 +147,16 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
       (panningMediaState) => this.onPanningMediaStateChange_(panningMediaState),
       true /** callToInitialize */
     );
+    this.storeService_.subscribe(
+      StateProperty.UI_STATE,
+      (uiState) =>
+        (this.minPageDistance_ = uiState === UIType.DESKTOP_PANELS ? 1 : 2),
```

Comment:
Nit: Since this is a lambda function, might be more readable if it's wrapped in `{}` instead of `()`